apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/sampling/SamplingService.java

```diff
@@ -109,7 +110,7 @@ private void resetSamplingFactor() {
     /**
      * Handle the samplingRate changed.
      */
-    void handleSamplingRateChanged() {
+    public void handleSamplingRateChanged() {
```

Comment:
Why change this?

Reply:
Considering there will be more `watcher`s, I moved to `org.apache.skywalking.apm.agent.core.conf.dynamic.watcher` package.

Reply:
I see.

Reply:
Considering there will be more `watcher`s, I moved to `org.apache.skywalking.apm.agent.core.conf.dynamic.watcher` package.